Assets/Scripts/VrSdk.cs

```diff
@@ -339,6 +350,13 @@ private void OnInputFocus(params object[] args)
             m_HasVrFocus = (bool)args[0];
         }
 
+#if OCULUS_SUPPORTED
+        private void DisplayRefreshRateChanged (float fromRefreshRate, float ToRefreshRate)
+        {
+            Debug.LogWarning(string.Format("Refresh rate changed from {0} to {1}", fromRefreshRate, ToRefreshRate));
```

Comment:
I don't think this needs to be a warning?

Reply:
This will also be called if it's changed without a specific request, e.g., thermal throttling. I figured it would be useful to know about it in these cases.